src/index.js

```diff
@@ -0,0 +1,38 @@
+import './main.css';
+import ToDo from './module/todo.js';
+import Storage from './module/localStorage.js';
+import updateDom from './module/dom.js';
+import { success, error } from './module/utilities.js';
+
+const input = document.querySelector('#todo-entry');
+let storage = [];
```

Comment:
- [ ] Please, add elements as the first elements on the project. Is important to have example elements to check the project. 🐶